<launch>
        <node name = "force_mcu_node" pkg = "robotswitch_controller" type = "force_mcu_node" respawn="false" output="screen" >
                <param name="force_port_"  value="/dev/ttyUSB3"/>
                <param name="force_baud_"  value="115200"/>
        </node>
</launch>